[2022-11-30T07:02:49.570+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: first_dag.world scheduled__2022-10-19T00:00:00+00:00 [queued]>
[2022-11-30T07:02:49.576+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: first_dag.world scheduled__2022-10-19T00:00:00+00:00 [queued]>
[2022-11-30T07:02:49.576+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-30T07:02:49.576+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-30T07:02:49.576+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-30T07:02:49.584+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): world> on 2022-10-19 00:00:00+00:00
[2022-11-30T07:02:49.587+0000] {standard_task_runner.py:55} INFO - Started process 2224 to run task
[2022-11-30T07:02:49.592+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'world', 'scheduled__2022-10-19T00:00:00+00:00', '--job-id', '593', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp1_p5w4qb']
[2022-11-30T07:02:49.592+0000] {standard_task_runner.py:83} INFO - Job 593: Subtask world
[2022-11-30T07:02:49.639+0000] {task_command.py:376} INFO - Running <TaskInstance: first_dag.world scheduled__2022-10-19T00:00:00+00:00 [running]> on host 733208eb192a
[2022-11-30T07:02:49.704+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=world
AIRFLOW_CTX_EXECUTION_DATE=2022-10-19T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-19T00:00:00+00:00
[2022-11-30T07:02:49.704+0000] {logging_mixin.py:137} INFO - Hello!
[2022-11-30T07:02:49.704+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-11-30T07:02:49.715+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=world, execution_date=20221019T000000, start_date=20221130T070249, end_date=20221130T070249
[2022-11-30T07:02:49.762+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-30T07:02:49.773+0000] {taskinstance.py:2623} INFO - 0 downstream tasks scheduled from follow-on schedule check
